yroots/subdivision.py

```diff
@@ -24,11 +24,13 @@
 import warnings
 from numba import jit
 
+macheps = 2.220446049250313e-16
+
 def solve(funcs, a, b, rel_approx_tol=1.e-15, abs_approx_tol=1.e-12,
           max_cond_num=1e5, good_zeros_factor=100, min_good_zeros_tol=1e-5,
           check_eval_error=True, check_eval_freq=1, plot=False,
           plot_intervals=False, deg=None, target_deg=None, max_level=999,
-          return_potentials=False, method='svd', target_tol=1.e-16):
+          return_potentials=False, method='svd', target_tol=macheps):
```

Comment:
```suggestion
          return_potentials=False, method='svd', target_tol=2*macheps):
```
Changing target_tol to be a little larger than macheps avoids problems when our approx errors and our tolerances end up being macheps once we've subdivided quite a bit. In my testing, I found that it doesn't have to be `2*macheps`, it could be `1.5*macheps` or even `1.01*macheps`, etc.